library/src/main/java/com/novoda/downloadmanager/InternalMigrationStatus.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.downloadmanager;
+
+interface InternalMigrationStatus extends MigrationStatus {
```

Comment:
Are you 100% you need an internal interface with one implementation?

Reply:
Yes, I can't imagine us requiring another.